docs/source/api_reference/featurizers.rst

```diff
@@ -472,6 +472,30 @@ DummyFeaturizer
   :members:
   :inherited-members:
 
+Tokenizers
+----------
+
```

Comment:
Can you add some descriptive text here explaining what a tokenizer is and why DeepChem uses them?

Reply:
moved grover tokenizers under the heading molecule tokenizers which contains description and need of the tokenizers.